EXAMPLES/src/WebpackEntry.res

```diff
@@ -1 +1,4 @@
-ReactDOMRe.renderToElementWithId(<App />, "root")
+switch ReactDOM.querySelector("#root") {
```

Comment:
For my own curiosity can you get into why this change is necessary here? Thanks!

Reply:
`ReactDOMRe` is no longer available in `@rescript/react`. I also switched to using the pattern of loading an app pulled directly from the documentation that uses `querySelector`. If I were building a real app, I'd probably at least log on the `None` case.

Reply:
Ah thanks! I missed that little update with the change!